src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/WrapWithTag/WrapWithTagEndpoint.cs

```diff
@@ -97,9 +99,11 @@ internal class WrapWithTagEndpoint : IWrapWithTagHandler
                 return null;
             }
 
+            cancellationToken.ThrowIfCancellationRequested();
+
             var parameter = request;
-            var response = await _languageServer.SendRequestAsync(LanguageServerConstants.RazorWrapWithTagEndpoint, parameter);
-            var htmlResponse = await response.Returning<WrapWithTagResponse>(cancellationToken);
+            var response = await _languageServer.SendRequestAsync(LanguageServerConstants.RazorWrapWithTagEndpoint, parameter).ConfigureAwait(false);
```

Comment:
`_languageServer.SendRequestAsync` really ought to take a `cancellationToken`. We'll get there some day I suppose.

Reply:
Lots of things in Razor should take a cancellation token :P